docs/workflow/analyze.md

```diff
@@ -1,20 +1,136 @@
-* Computing the PCA of the correspondence models and animating the variation of shape along with a principal component.
-* Visualization of the mean shape.
-* Exports the eigenvalues and the PCA loadings of each shape in an excel sheet for analysis.
-* If there are groups in the data, it can visualize group differences.
-
-To execute the *ShapeWorksStudio* GUI, it takes in an *XML* parameter file which is generated by the *`ShapeWorksRun`*
-in the optimization stage, called *`analyze.xml`*, it includes the path to the input data and corresponding particles.
- The following command executes the GUI.
-  
+## Surface Correspondences 
+
+ShapeWorks includes a Qt and VTK-based graphical user interface (GUI), **ShapeWorksStudio**, that allows visualizing the optimized particle system (i.e., correspondence model) for each shape sample where particle coloring is used to reflect correspondence among shapes. 
+
+
+*Particles' coloring reflects surface correspondences across different shape samples*
+![Template-deformation approach for surface reconstruction](../img/workflow/corres.png)
+
+
+You can scroll through the dataset and zoom in and out to inspect fewer or more samples. ShapeWorksStudio keeps a consistent camera view across all shape samples to facilitate qualitative comparisons of different samples relative to each other.
+
+*Correspondence model inspectation by scrolling though the dataset and zooming in & out*
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_scroll.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+## Running ShapeWorksStudio
+
+When you open ShapeWorksStudio without a project, either from terminal or double-clicking on the application binary/exe file, the splash screen is displayed to enable you to load a recent project, open existing projects on your local machine, or create a new project.
+
+*ShapeWorksStudio splash screen*
+![Template-deformation approach for surface reconstruction](../img/workflow/studio_splash.png)
+
+### Using XML Files
+
+You can run ShapeWorksStudio using an *XML* parameter file that includes the following tags.
+
+- `<point_files>` [required]: list of `_world.particles` files (output of the `shapeworks optimize`)
+- `<group_ids>` [optional]: the group id (1 or 2) in case the data contains groups
+- `<world_point_files>` [optional]: list of `_world.particles` files (output of the `shapeworks optimize`), same as `<point_files>`
+- `<local_point_files>` [optional]: list of `_local.particles` files (output of the `shapeworks optimize`)
+- `<distance_transform_files>` [optional]: list of distance transforms (input of the `shapeworks optimize`)
+
+
+Please note the following:
+
+- If the xml file only contains the `<point_files>` list, VTK-based surface reconstruction is used for surface reconstruction. 
+- To enable particle-based surface reconstruction,`<world_point_files>`, `<local_point_files>`, and `<distance_transform_files>` lists should be given.
```

Comment:
This is more of a general question, but why do we have the <world_point_files> tag if it is always the same as <point_files>?

Reply:
Good point. I think it was included as-is from the parameter file of the reconstruction command-line tool. But this should be cleaned up. I will submit an issue.